### ISSUE TITLE
fix(security): address CVE-2021-23337

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#1778]
https://github.com/advisories/GHSA-35jh-r3h4-6jhm

**Comment by petermetz** [comment]
@aldousalvarez You got it, thanks for the help!

**Comment by aldousalvarez** [comment]
Hi @petermetz This are the packages that are using the web3-eea that has its dependencies on lodash that is using the version 4.17.19 which are vulnerable to command injection and needs to be upgraded to 4.17.21 

cactus-plugin-htlc-eth-besu, 
cactus-plugin-ledger-connector-besu, 
cactus-test-plugin-htlc-eth-besu, 
cactus-test-plugin-ledger-connector-besu 

and web3-eea is now deprecated so in order to update the lodash to the latest version I removed the web3-eea and installed the web3js-quorum and followed the steps migrating it and there are files that are dependent on web3-eea so I applied the changes and migrated them because it is almost the same but the methods are just renamed or moved into different namespaces and parameters but when I finished the migration and I am testing the files it says invalid params and the error is found when calling the priv.call function.

these are the files that are using the priv.call function
1. packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts
2. packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/private-deploy-contract-from-json-cactus.test.ts
3. packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/private-deploy-contract-from-json-web3-eea.test.ts

this is the priv.call function of web3js-quorum
consensys.github.io/web3js-quorum/latest/module-priv.html#~call

this is the priv.call function of web3-eea
https://consensys.github.io/web3js-eea/latest/EEAClient.html#~call

here is the link of the migration: 
consensys.github.io/web3js-quorum/latest/tutorial-Migrate%20from%20web3js-eea.html

here is the index.d.ts file that I used: 
github.com/ConsenSys/web3js-quorum/blob/master/src/typescript/index.d.ts
